draft-ietf-scone-protocol.md

```diff
@@ -612,78 +612,80 @@ To reduce the risk of synchronization across multiple senders,
 
 When advice is set or updated,
-the network element waits until
-it receives the next SCONE packet on affected flows.
-It then updates the throughput advice in that packet ({{apply}}).
+monitoring cannot start until the network element
+transmits a SCONE packet with
+throughput advice other than `Unknown` (127); see {{apply}}.
 
 A network element can then monitor affected flows
 to determine whether the throughput advice it provided
 was followed.
 
 A network element SHOULD base its monitoring
-on the maximum value it provided for throughput advice
+on the maximum value it signaled for throughput advice
 over at least the preceding monitoring period.
-Additional time might be allowed
-to account for the possibility that some SCONE packets might be lost.
+A longer period might be used
+to account for the possibility that SCONE packets are lost.
 This allows applications time to receive advice
 and adapt their sending rate.
 
-A network element MUST NOT alter datagrams to add SCONE packets
-or synthesize datagrams that contain SCONE packets.
-The latter will not be accepted and the former,
-even if they do not exceed the path MTU as a result,
-can be detected by applications and could be ignored.
-This document does not define a mechanism to support detection,
-but one might be added in future.
-
 
 ## Flows That Exceed Throughput Advice {#policing}
 
-Network elements that provide throughput advice
-can monitor flows --
-or sets of flows that are subject to the same throughput limit --
-for adherance to that advice.
-
-In the event that a flow exceeds these limits,
-a network element could immediately start
-enforcing adherence to the advice as though it were a hard limit.
-However, this risks creating a situation where communication ceases entirely
-for a significant period of time;
-that is, up to the period defined in {{time}}.
-
-A better approach is to exclude any data transmitted
-before sending reduced throughput advice
-from any monitoring that uses the reduced rate.
-This ensures that the enforcement of limits is minimally disruptive.
+A network could deploy policy enforcement that drops or delays packets
+to ensure that applications do not exceed throughput limits set in policy.
+
+SCONE allows networks to provide advice to applications,
+so that stricter limits,
+which can be inefficient and lead to worse application performance,
+are not necessary in all cases.
+
+Some applications will not support SCONE.
+Other applications either will not
+or cannot
+follow throughput advice.
+
+Networks can monitor flows to determine if applications follow advice;
+see {{monitoring}}.
+A network could choose to either disable or loosen policy enforcement
+for flows where SCONE is active,
+but re-enable or tighten enforcement if monitoring indicates
+that throughput advice is not being respected.
+
+Any monitoring and policy enforcement could be implemented
+in different network elements than the ones that signal throughput advice.
```

Comment:
It might make sense to change like below, to avoid the impression that monitoring and enforcement can be done base on untrusted rate signals (#98):
```suggestion
Monitoring and policy enforcement based on network policy could be implemented
in different network elements than the ones that signal throughput advice.
```

Though, separately, I think this paragraph might need to be moved to #monitoring.